### PR TITLE
Add default to ExportProcessorType docstrings

### PR DESCRIPTION
**Pull request by benbhall**
## Changes

Added default ExportProcessorType (Batch) to the docstrings for:

- Zipkin Exporter
- Jaeger Exporter
- OpenTelemetry Protocol Exporter

### CODE REVIEW

**Review by cijothomas — APPROVED** [review]
thanks.

File: src/OpenTelemetry.Exporter.Jaeger/JaegerExporterOptions.cs [hunk 1]
```diff
@@ -75,7 +75,7 @@ public JaegerExporterOptions()
         public int? MaxPayloadSizeInBytes { get; set; } = DefaultMaxPayloadSizeInBytes;
 
         /// <summary>
-        /// Gets or sets the export processor type to be used with Jaeger Exporter.
+        /// Gets or sets the export processor type to be used with Jaeger Exporter. The default value is Batch.
```
Comment (reyang) [inline src/OpenTelemetry.Exporter.Jaeger/JaegerExporterOptions.cs:78]: Is there a way to make it a [cref](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/cref-attribute)?

**Review by reyang — APPROVED** [review]

File: src/OpenTelemetry.Exporter.Jaeger/JaegerExporterOptions.cs (hunk 1 above)
Comment (cijothomas) [reply src/OpenTelemetry.Exporter.Jaeger/JaegerExporterOptions.cs:78]: Should be.. @benbhall Could you do that change?

Comment (benbhall) [reply src/OpenTelemetry.Exporter.Jaeger/JaegerExporterOptions.cs:78]: Yep, on it.

**Review by cijothomas — APPROVED** [review]

**Pull request merged**